vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -140,13 +140,19 @@ $global:map = @{
         "https://acs-mirror.azureedge.net/kubernetes/v1.24.3-hotfix.20221006/windowszip/v1.24.3-hotfix.20221006-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.24.6-hotfix.20221006/windowszip/v1.24.6-hotfix.20221006-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.24.9/windowszip/v1.24.9-1int.zip",
```

Comment:
do we need to remove `https://acs-mirror.azureedge.net/kubernetes/v1.24.9/windowszip/v1.24.9-1int.zip` ?

Reply:
Not sure, but Principle #3 says we need to keep one old version